plugins/via-usbhub/meson.build

```diff
@@ -0,0 +1,53 @@
+cargs = ['-DG_LOG_DOMAIN="FuPluginViaUsbhub"']
+
+install_data(['via-usbhub.quirk'],
+  install_dir: join_paths(datadir, 'fwupd', 'quirks.d')
+)
+
+shared_module('fu_plugin_via_usbhub',
+  fu_hash,
+  sources : [
+    'fu-plugin-via-usbhub.c',
+    'fu-via-usbhub-common.c',
+    'fu-via-usbhub-device.c',
+    'fu-via-usbhub-firmware.c',
+  ],
+  include_directories : [
+    include_directories('../..'),
+    include_directories('../../src'),
+    include_directories('../../libfwupd'),
+  ],
+  install : true,
+  install_dir: plugin_dir,
+  link_with : [
+    libfwupdprivate,
+  ],
+  c_args : cargs,
+  dependencies : [
+    plugin_deps,
+  ],
+)
+
+if get_option('tests')
+  # for fuzzing
+  executable(
```

Comment:
probably should mention this in top level readme too

Reply:
So, I'm not sure if to include this utility or not. It's impossible to make "small" firmware files the fuzzer can do anything sensible with, due to the large fixed offsets the parser does. I'll play with afl and see if I can convince it to do something sane.

Reply:
Now we have 95c98a95b6faae6b710dacb88f831913fa1ed5a1 we don't need this :)